sycl/doc/extensions/experimental/sycl_ext_oneapi_non_uniform_groups.asciidoc

```diff
@@ -327,44 +340,31 @@ public:
 }
 ----
 
-NOTE: `ballot_group` is templated on a `ParentGroup` because it is expected
-that it will eventually be possible to construct a ballot-group from more
-than only sub-groups.
-
 [source,c++]
 ----
 id_type get_group_id() const;
 ----
-_Returns_: An `id` representing the index of the ballot-group.
-
-NOTE: This will always be either 1 (representing the group of work-items where
-the predicate was true) or 0 (representing the group of work-items where the
-predicate was false).
+_Returns_: An `id` representing the index of the fragment within its parent.
 
 [source,c++]
 ----
 id_type get_local_id() const;
 ----
-_Returns_: An `id` representing the calling work-item's position within
-the ballot-group.
+_Returns_: An `id` representing the calling work-item's position within the
+fragment.
 
 [source,c++]
 ----
 range_type get_group_range() const;
 ----
-_Returns_: A `range` representing the number of ballot-groups.
-
-NOTE: This will always return a `range` of 2, as there will always be two groups;
-one representing the group of work-items where the predicate was true and
-another representing the group of work-items where the predicate was false.
-It is possible for one of these groups to contain no work-items, if the
-predicate has the same value on all work-items.
+_Returns_: A `range` representing the number of fragments created when the
+parent group was partitioned.
 
 [source,c++]
 ----
 range_type get_local_range() const;
 ----
-_Returns_: A `range` representing the number of work-items in the ballot-group.
+_Returns_: A `range` representing the number of work-items in the fragment.
```

Comment:
Can't comment there, but some of the descriptions below should be updated as well (see those fixes proposed in #13702):
- `get_local_liner_id` description is recursive
- `get_group_linear_range` and `get_local_linear_range` should return `linear_id_type`
- `get_group_linear_range` and `get_local_linear_range` descriptions should be "`range` returned by" instead of "`id` returned by"

Reply:
Fixed in cc24795326e7da667c927e701ec472e93ec67d6b.